osquery/carver/carver.cpp

```diff
@@ -318,4 +313,29 @@ Status Carver::postCarve(const boost::filesystem::path& path) {
   updateCarveValue(carveGuid_, "status", "SUCCESS");
   return Status(0, "Ok");
 };
+
+Status carvePaths(const std::set<std::string>& paths) {
+  auto guid = generateNewUUID();
+  pt::ptree tree;
+  tree.put("carve_guid", guid);
+  tree.put("time", getUnixTime());
+  tree.put("status", "STARTING");
+  tree.put("sha256", "");
+  tree.put("size", -1);
+  if (paths.size() > 1) {
+    tree.put("path", boost::algorithm::join(paths, ","));
+  } else {
+    tree.put("path", *(paths.begin()));
+  }
+  std::ostringstream os;
```

Comment:
This code was moved and the spaces were left out, IMO a double newline goes a long way for readability.